src/transformers/models/fuyu/image_processing_fuyu.py

```diff
@@ -59,125 +236,345 @@ class FuyuImageProcessor(BaseImageProcessor):
 
     def __init__(
         self,
-        patch_size: Dict[str, int] = None,
-        target_height: int = 1080,
-        target_width: int = 1920,
+        do_resize: bool = True,
+        size: Optional[Dict[str, int]] = None,
+        resample: PILImageResampling = PILImageResampling.BILINEAR,  # FIXME check default value
```

Comment:
```suggestion
        resample: PILImageResampling = PILImageResampling.BILINEAR, 
```

This is how it was done in the original code: https://huggingface.co/adept-hf-collab/adept-mm/blob/736c6b570b2a9c0367a3266746fd1f53cfff0a2b/mm-inference-for-hf/multimodal/data/image_utils.py#L208

`BILINEAR` seems correct, as our resizing is always done on PIL images and `antialias` is True in that case.